README.md

```diff
@@ -40,7 +40,8 @@ The [OCI Image Media Types](media-types.md) document is a starting point to unde
 The high-level components of the spec include:
 
 * An archival format for container images, consisting of an [image manifest](manifest.md), an [image layout](image-layout.md), a set of [filesystem layers](layer.md), and [image configuration](config.md) (base OCI layer)
-* A process of hashing the image format for integrity and content-addressing (base OCI layer)
+* A [process of hashing the image format](descriptor.md) for integrity and content-addressing (base OCI layer)
+* A format for [storing CAS blobs and references to them](image-layout.md) (optional OCI layer)
```

Comment:
As thrilled as I would be for the scope table to be updated I can assure you that will be considerably more involved than just tweaking this PR. IMO image layout is basically covered in the first bullet point in this list - it's essentially working towards the "Archival Format" (see ["longer term"](https://github.com/opencontainers/image-spec/pull/94#issue-157118241).